### DISCUSSION TITLE
Endpoint findAllListingsActive returns inventories with other value_ids than endpoint getPropertiesByTaxonomyId.

### DISCUSSION THREAD

**Discussion by BenjaminAtBillbee** [etsy/open-api#918] [Q&A]
Example:
- findAllListingsActive returns value_id 50831750264 for "Primary color" "Beige".
- getPropertiesByTaxonomyId returns value_id 1213 for  "Primary color" "Beige".

Question:
How to map from a value_id to its meaning? What is the correct value_id for "Primary Color" "Beige"?

Details:

findAllListingsActive returns:
```
"inventory": {
        "products": [
          {
            "product_id": 11185478599,
            "sku": "",
            "is_deleted": false,
            "offerings": [
              {
                "offering_id": 11302409279,
                "quantity": 8,
                "is_enabled": true,
                "is_deleted": false,
                "price": {
                  "amount": 200,
                  "divisor": 100,
                  "currency_code": "EUR"
                }
              }
            ],
            "property_values": [
              {
                "property_id": 200,
                "property_name": "Primary color",
                "scale_id": null,
                "scale_name": null,
                "value_ids": [
                  50831750264
                ],
                "values": [
                  "Beige"
                ]
              }
            ]
          },

```

getPropertiesByTaxonomyId returns:
```
"results": [
        {
            "property_id": 200,
            "name": "Primary color",
            "display_name": "Primary color",
            "scales": [],
            "is_required": false,
            "supports_attributes": true,
            "supports_variations": true,
            "is_multivalued": false,
            "max_values_allowed": 5,
            "possible_values": [
                {
                    "value_id": 1213,
                    "name": "Beige",
                    "scale_id": null,
                    "equal_to": []
                },
```

**Comment by devtronic** [comment]
@etsyachristensen can your team create a mapping file between the ids so we can map it in our code manually? We require a solution to migrate our users from v2 to v3. Without a unique mapping our users are forced to edit all their products and add a sku to them.

**Reply by PMStanley** [reply]
Andrew left in December 2022: https://github.com/etsy/open-api/discussions/768

His leaving post suggested 4 people were stepping in to start answering questions here and clear down the open discussions/issues but I'm not sure that's happened and you may well be better off emailing the support team directly (and updating the post here if you get an answer)
